components/ExternalDa.vue

```diff
@@ -102,14 +101,12 @@ let shownFirst = false
 
 const emit = defineEmits(['rendered'])
 
-async function visibilityChanged(visible) {
+function visibilityChanged(visible) {
   try {
     if (visible && !shownFirst) {
       isVisible.value = visible
       shownFirst = true
 
-      await waitForRef(uniqueid)
```

Comment:
I'm not sure about removing this one.  `visibilityChanged` will get called, then we'll set `isVisible`.  That will cause ref `adUnitPath` to get into the DOM, but I don't see any reason why we can assume that this will have happened before the Google code tries to access that element.  Don't we need an `await nextTick()`?